### DISCUSSION TITLE
Buildings with squared corners in Openstreetmap rendered not squared in OSMand

### DISCUSSION THREAD

**Discussion by petenz** [osmandapp/OsmAnd#12368] [Maps Related]
I contribute to Openstreetmap regularly, often adding buildings such as hiking huts etc in remote areas. I use the 'square corners' tool to make buildings correct but they seem to render incorrectly in OSMand when they turn up there. What could be the cause of this?

**Comment by tommycrock** [comment]
This has always happened in osmand (at least for me). I assume that
positions get rounded in the process of reducing the database to a portable
size. I've seen some micromapped nodes get removed also. If someone else
knows better I'd be interested too.

**Reply by scaidermern** [reply]
Yes, this has always been the case.

I think this is the same issue as #6130.
